plugins/web-worker-offloading/third-party/google-site-kit.php

```diff
@@ -0,0 +1,62 @@
+<?php
+/**
+ * Web Worker Offloading integration with Site Kit by Google.
+ *
+ * @since n.e.x.t
+ * @package web-worker-offloading
+ */
+
+if ( ! defined( 'ABSPATH' ) ) {
+	exit; // Exit if accessed directly.
+}
+
+/**
+ * Configures WWO for Site Kit and Google Analytics.
+ *
+ * @since n.e.x.t
+ * @access private
+ * @link https://partytown.builder.io/google-tag-manager#forward-events
+ *
+ * @param array<string, mixed>|mixed $configuration Configuration.
+ * @return array<string, mixed> Configuration.
+ */
+function plwwo_google_site_kit_configure( $configuration ): array {
+	$configuration = (array) $configuration;
+
+	$configuration['globalFns'][] = 'gtag'; // Because gtag() is defined in one script and called in another.
+	$configuration['forward'][]   = 'dataLayer.push'; // Because the Partytown integration has this in its example config.
```

Comment:
Per https://github.com/WordPress/performance/pull/1685/files#r1884327683:

```suggestion
	$configuration['forward'][]   = 'dataLayer.push'; // See <https://partytown.builder.io/forwarding-event>.
```